<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWin{
color: rgb(255, 245, 235);
background-color: rgb(34, 33, 33);
border: 2px solid rgb(42, 41, 39);
border-radius: 5px ;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="fileNameLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
color:rgb(69, 67, 65);
	font: 9pt &quot;JetBrains Mono&quot;;
}</string>
        </property>
        <property name="text">
         <string>File*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="CodeEditor" name="plainTextEdit">
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit {
border: 2px solid rgb(42, 41, 39);
border-radius: 5px;
color: rgb(255, 245, 235);
background-color: rgb(34, 33, 33);
font: 9pt &quot;JetBrains Mono&quot;;
height: 0px;
width:0px;
selection-background-color: rgb(250, 228, 176);
selection-color:rgb(42, 41, 39);
selection-radius: 2px
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
color:rgb(69, 67, 65);
	font: 9pt &quot;JetBrains Mono&quot;;
}</string>
        </property>
        <property name="text">
         <string>Memory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="registersLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
color:rgb(69, 67, 65);
	font: 9pt &quot;JetBrains Mono&quot;;
}</string>
        </property>
        <property name="text">
         <string>Registers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QTableView" name="registerView">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView{
border: 2px solid rgb(42, 41, 39);
border-radius: 5px;
color:rgb(103, 100, 97);
background-color: rgb(34, 33, 33);
font: 9pt &quot;JetBrains Mono&quot; ;
height: 0px;
width:0px;
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="QTableView" name="tableView">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView{
border: 2px solid rgb(42, 41, 39);
border-radius: 5px;
color:rgb(103, 100, 97);
background-color: rgb(34, 33, 33);
font: 9pt &quot;JetBrains Mono&quot; ;
height: 0px;
width:0px;
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="topBarWidget" name="topBar" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#topBar{
background-color: rgb(42, 41, 39);
border: 0px solid;
border-top-left-radius: 20px;
border-top-right-radius: 20px;
border-bottom-right-radius: 20px;
border-bottom-left-radius: 20px;
border-color:rgb(42, 41, 39);
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="4">
        <widget class="QPushButton" name="fileButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 9pt &quot;JetBrains Mono&quot;;
	color:rgb(103, 100, 97)

}</string>
         </property>
         <property name="text">
          <string>file</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" rowspan="2" colspan="2">
        <widget class="QLabel" name="iconLabel">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/resources/RAWRS.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="maximizeButton">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/resources/MAXIM.png</normaloff>:/resources/MAXIM.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QPushButton" name="exitButton">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/resources/EXIT.png</normaloff>:/resources/EXIT.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="executeButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 9pt &quot;JetBrains Mono&quot;;
	color:rgb(103, 100, 97)

}</string>
         </property>
         <property name="text">
          <string>execute</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="minimizeButton">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/resources/MINIM.png</normaloff>:/resources/MINIM.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>iconLabel</zorder>
      <zorder>minimizeButton</zorder>
      <zorder>maximizeButton</zorder>
      <zorder>executeButton</zorder>
      <zorder>exitButton</zorder>
      <zorder>fileButton</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1275</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>topBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">topbarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
